scripts/build_docker.sh

```diff
@@ -14,6 +14,10 @@ then
     docker tag grafana/metrictank grafana/metrictank:$tag
     docker tag grafana/metrictank grafana/metrictank:$version
 
+    docker build -t grafana/mt-gateway --target=mt-gateway -f ${DIR}/Dockerfile .
+    docker tag grafana/mt-gateway grafana/mt-gateway:$tag
+    docker tag grafana/mt-gateway grafana/mt-gateway:$version
```

Comment:
Is `mt-gateway` descriptive enough for the docker image name?

`grafana/metrictank-gateway` maybe?

Reply:
i think mt-gateway is fine. people will only come looking for this when they already looked into metrictank, and were linked to this.
our target users are not people who randomly browse around the internet, find this and say "hey i want to use this but i don't know what mt means"

Reply:
Do I need to have someone create the initial repository, or does that just automatically happen on push?

Would be nice to not have the build break on merge

Reply:
should work. we'll see